lib/rules/no-extra-parens.js

```diff
@@ -574,15 +575,13 @@ module.exports = {
                              * If `let` is the only thing on the left side of the loop, it's the loop variable: `for ((let) of foo);`
                              * Removing it will cause a syntax error, because it will be parsed as the start of a VariableDeclarator.
                              */
-                            firstLeftToken.range[1] === node.left.range[1] ||
-
-                            /*
+                            (firstLeftToken.range[1] === node.left.range[1] || /*
```

Comment:
Was this change intentional? (The parentheses is probably fine but the comment should probably begin on the next line.)

Reply:
Ironic, given the rule this is in 🤣